src/Nancy.Testing/Browser.cs

```diff
@@ -219,7 +219,9 @@ public BrowserResponse HandleRequest(string method, Url url, Action<BrowserConte
             var request =
                 CreateRequest(method, url, browserContextValues);
 
-            var response = new BrowserResponse(this.engine.HandleRequest(request), this, (BrowserContext)browserContextValues);
+            var context = await this.engine.HandleRequest(request);

```

Comment:
already made that change, see last commit
On 5 Dec 2015 3:31 pm, "Steven Robbins" <notifications@github.com> wrote:

> In src/Nancy.Testing/Browser.cs
> <https://github.com/NancyFx/Nancy/pull/2105#discussion_r46758848>:
>
> > @@ -219,7 +219,9 @@ public BrowserResponse HandleRequest(string method, Url url, Action<BrowserConte
> >              var request =
> >                  CreateRequest(method, url, browserContextValues);
> >
> > -            var response = new BrowserResponse(this.engine.HandleRequest(request), this, (BrowserContext)browserContextValues);
> > +            var context = await this.engine.HandleRequest(request);
>
> Yep ________________________________ From: Damian Hickey<mailto:
> notifications@github.com> Sent: ‎05/‎12/‎2015 12:54 To:
> NancyFx/Nancy<mailto:Nancy@noreply.github.com> Subject: Re: [Nancy] Using
> async/await (#2105 <https://github.com/NancyFx/Nancy/pull/2105>)
> @@ -219,7 +219,9 @@ public BrowserResponse HandleRequest(string method,
> Url url, Action<BrowserConte var request = CreateRequest(method, url,
> browserContextValues); - var response = new
> BrowserResponse(this.engine.HandleRequest(request), this,
> (BrowserContext)browserContextValues); + var context = await
> this.engine.HandleRequest(request);
> **Question**: anywhere we do an await within Nancy, should we also
> `.ConfigureAwait(false)`. My understanding is that all libraries should be
> doing that.
> … <#151728dba792cc8b_>
> --- Reply to this email directly or view it on GitHub:
> https://github.com/NancyFx/Nancy/pull/2105/files#r46757977
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NancyFx/Nancy/pull/2105/files#r46758848>.
>